client/blocks/signup-form/index.jsx

```diff
@@ -622,7 +623,28 @@ class SignupForm extends Component {
 												onClick={ ( event ) => this.handleLoginClick( event, fieldValue ) }
 											/>
 										),
-										pwdResetLink: <a href={ lostPassword( this.props.locale ) } />,
+										pwdResetLink: isReactLostPasswordScreenEnabled() ? (
+											<a
+												href="/"
```

Comment:
Small accessibility comment - some screenreaders will pick up this href and send the user there instead. Shall we continue using the old non-react url as a fallback?

Reply:
@rjchow good call! Updated it in [Use legacy lost password link as a fallback](https://github.com/Automattic/wp-calypso/pull/95002/commits/743357d7bc03bb3209f529fdd70be25dd4ae2c7e)